bi-connectors/PowerBIConnector/OpenSearch.md

```diff
@@ -0,0 +1,67 @@
+#  Connecting OpenSearch to Microsoft Power BI Desktop
+
+>[!Note]
+
+>The following connector article is provided by OpenSearch, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the OpenSearch website and use the support channels there.
+
+## Summary
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI Desktop |
+| Authentication Types Supported | Basic |
+| | |
+
+## Prerequisites
+* Microsoft Power BI Desktop
+* [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
+
+## Capabilities supported
+* Import
+* DirectQuery
+
+## Connect to OpenSearch
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **OpenSearch**. Click on **Connect**.

```

Comment:
Should this deal with installing the connector? What about enabling uncertified connectors?

Reply:
I don't believe so as the user shouldn't have to install the connector once the connector is certified. For example [Azure](https://github.com/MicrosoftDocs/powerquery-docs/blob/main/powerquery-docs/Connectors/Azure%20Cosmos%20DB%20v2.md) does not include installation of the connector.

Reply:
Ack - so the intent is this is for the certified connector.